docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_sequence.md

```diff
@@ -30,6 +30,11 @@ The sequence name must be distinct from any other sequences, tables, indexes, vi
 
 ### *sequence_options*
 
+#### AS *seq_data_type*
+
+Specifies the data type of the sequence. This automatically determines the minimum and maximum values of the sequence. Valid types are `smallint`, `integer`, and `bigint`. Default is `bigint`.
+
+
 #### TEMPORARY or TEMP
```

Comment:
should TEMP be under sequence options? Should go first at least?

Reply:
You're right, changed in https://github.com/yugabyte/yugabyte-db/pull/21798/commits/fca2d0f56e8ddd6e79cbba1d28c41202bb592868